lib/Map/AddressConverter.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/*global require*/
+var DeveloperError = require('terriajs-cesium/Source/Core/DeveloperError');
+
+/*
+Determine lat and long coordinates for a given address.
+If TableStructure has address column, adds two new columns: lat and long.
+*/
+
+/**
+ * Instantiate an address converter with a TableStructure that it must convert.
```

Comment:
This isn't true is it?  The `tableStructure` is passed to `bulkConvertAddresses`, it's not passed to the constructor.